src/app/containers/Metadata/linkedDataAbout.js

```diff
@@ -0,0 +1,36 @@
+const acceptableTypes = ['Person', 'Event', 'Organization', 'Place'];
+
+export const checkType = types => {
+  if (types.length === 0 || types.length > 1) {
+    return 'Thing';
+  }
+  return acceptableTypes.includes(types[0]) ? types[0] : 'Thing';
+};
+
+export const checkSameAs = uris => {
+  const sameAs = uris.filter(uri => uri.includes('http://dbpedia.org'));
+  return sameAs.length ? sameAs : undefined;
+};
+
+const aboutTagsContent = aboutTags => {
+  if (aboutTags && aboutTags.length > 0) {
+    const content = [];
+    aboutTags.forEach(tag => {
+      const about = {
+        '@type': checkType(tag.thingType),
+        name: tag.thingLabel,
+        alternateName: tag['skos:altLabel'] ? tag['skos:altLabel'] : '',
```

Comment:
Reading through the requirements in the issue, I noticed that `alternateName` should only be added if there is a value for it. So can you please update this so that it doesn't include `alternateName` in the `about` object if it is not available?